config/pepperConfig.js.ctmpl

```diff
@@ -140,14 +140,19 @@ var DDP_ENV = {
 {{end}}
 
 {{if eq $study_guid "atcp"}}
+   {{with secret (printf "secret/pepper/%s/%s/%s/conf" $environment $version $study) }}
+        DDP_ENV['recaptchaSiteClientKey'] = "{{.Data.recaptchaSiteClientKey}}",
+        DDP_ENV['auth0Domain'] = "{{.Data.auth0.domain}}",
+   {{end}}
+
+   DDP_ENV['darUrl'] = "https://us-central1-broad-ddp-dev.cloudfunctions.net/cf-study-contact";
```

Comment:
Should probably put this into Vault. @YuriyPavlovBI Should this change to `...cloudfunctions.net/cf-atcp-contact` instead due to my refactor in https://github.com/broadinstitute/ddp-study-server/pull/633?

Reply:
@aleksey-kuznetsov Please switch to using
```
https://us-central1-broad-ddp-dev.cloudfunctions.net/cf-atcp-contact
```
It should already be available on `dev` environment.